EMML/1-1000/EMML151.xml

```diff
@@ -46,7 +46,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                         </support>
                         <extent>
                            <measure unit="leaf">ff 134+1</measure>
-                           <dimensions type="outer"unit="mm">
+                           <dimensions type="outer "unit="mm">
```

Comment:
sorry, the space should be AFTER the quotation mark, between the two attributes

type="outer" unit="mm"